docs/cicd.md

```diff
@@ -22,12 +22,12 @@ If no branch is specified, main is used as default. Per default the delivery is
 ## Workflows
 
 Several workflows exist for different purposes:
-* **commit**: Run for all commits of PRs to ensure good code quality and working code. Delivers Integreat and Malte webapps on the main branch.
-* **scheduled_native_development_delivery**: Delivers Integreat and Malte native builds to development (Testflight and Beta channel)
+* **commit**: Run for all commits of PRs to ensure good code quality and working code. Delivers Integreat, Malte and AschaffenburgApp webapps on the main branch.
```

Comment:
I think we should just stick with `Aschaffenburg`

Reply:
Agree

Reply:
done